### PR TITLE
AI Thinker: Fix volume levels

### PR DESCRIPTION
**Pull request by alexyao2015**
Follow up to #67 . I found the entire bottom half of the range is completely inaudible. This adjusts the range to only the top 50% of what it used to be. 

The changes to DACCONTROL24-27 appear to not be important to change.

### CODE REVIEW

File: components/audio_hal/driver/es8388/es8388.c [hunk 1]
```diff
@@ -346,6 +346,10 @@ es8388_init (audio_hal_codec_config_t *cfg)
                        0x80); // set internal ADC and DAC use the same LRCK
                               // clock, ADC LRCK as internal LRCK
   res |= es_write_reg (ES8388_ADDR, ES8388_DACCONTROL23, 0x00); // vroi=0
+  res |= es_write_reg (ES8388_ADDR, ES8388_DACCONTROL24, 0x1E); // Set L1 R1 L2 R2 volume. 0x00: -30dB, 0x1E: 0dB, 0x21: 3dB
```
Comment (CarlosDerSeher) [inline components/audio_hal/driver/es8388/es8388.c:349]: How does this impact other boards using the same codec (lyrat v4.3, etc.)?

Comment (alexyao2015) [reply components/audio_hal/driver/es8388/es8388.c:349]: They probably should be fine with this but I'm not sure since I don't have one.

Comment (CarlosDerSeher) [reply components/audio_hal/driver/es8388/es8388.c:349]: Ok I'll test this and compare

**Comment by unknown0816** [comment]
fyi: With this settings my Board is way to loud. I had to adapt this like written in this PR https://github.com/CarlosDerSeher/snapclient/pull/67#issuecomment-1975867355

**Comment by CarlosDerSeher** [comment]
Ok thank you, I will consider this in #69

**Comment by CarlosDerSeher** [comment]
So we should differentiate  between boards in the driver in your opinion? Could you please open an issue on this so I don't forget about it?

**Pull request merged**